orchestrator-service/internal/broker/kafka.go

```
@@ -0,0 +1,73 @@
+package broker
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/segmentio/kafka-go"
+)
+
+type broker struct {
+	conn *kafka.Conn
+}
+
+func New(c *kafka.Conn) *broker {
+	return &broker{
+		conn: c,
+	}
+}
+
+type Message struct {
+	Link string `json:"link"`
+}
+
+func (b *broker) Write(mes []byte) error {
+	_, err := b.conn.Write(mes)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Why not just to return an err?